molecule/default/molecule.yml

```diff
@@ -4,10 +4,12 @@ dependency:
   name: galaxy
 driver:
   name: docker
-lint:
-  name: yamllint
-  options:
-    config-file: molecule/default/yamllint.yml
+lint: |
+  E=0
+  yamllint -c molecule/default/yamllint.yml . || E=$$?
+  ansible-lint || E=$$?
+  flake8 || E=$$?
```

Comment:
same with `flake8`: `flake8 --exclude=.venv,.tox --statistics .` (--statistics gives a nice summary)

Reply:
Also - for me - this still doesn't work - I had to add `ansible-lint` and `flake8` to `molecule_extra_requirements.txt` - I suppose you could also put them in `tox.ini` in the `molecule_common` section `deps`

Reply:
I finally use only `yamllint` and `ansible-lint`. `flake8` is run by `tox` so imho it is not necessary to run `flake8` twice.